src/lib.rs

```diff
@@ -234,7 +234,7 @@ trait GlObject {
     fn get_id(&self) -> gl::types::GLuint;
 }
 
-/// Internal trait for enums that can be turned into GLenum.
+/// Internal trait for enums that can be turned into GLEnum.
```

Comment:
That should remain `GLenum`, as it's the name of the type (see three lines below).